### PR TITLE
docs(en): add en operation doc

### PR DESCRIPTION
**Pull request by palering**
**What this PR does**:  translate the operation guide into english

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: docs/en/operation/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+## 1. Deploy Layotto
+
+There are some ways to deploy Layotto that you can find below.
+
+- Deploy Layotto from release binaries
+- Deploy using Docker
+- Deploy on Kubernetes
+
+### Deploy Layotto from release binaries
```
Comment (seeflood) [inline docs/en/operation/README.md:11]: Deploy Layotto using released binaries

File: docs/en/operation/README.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+To run, download or build the binary for your platform, then exec it. Refer to the [Quick start](en/start) guide.
```
Comment (seeflood) [inline docs/en/operation/README.md:13]: 这句话看起来不通顺？

File: docs/en/operation/README.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+### Deploy using Docker
+
+You can run Layotto using the official Docker images.Currently include:
+
+- [layotto/layotto](https://hub.docker.com/repository/docker/layotto/layotto)
+- [layotto/layotto.arm64](https://hub.docker.com/repository/docker/layotto/layotto.arm64)
+
+It's not contain a `config.json` configuration file in the image, you can mount your own configuration file into the `/runtime/configs/` directory of the image. For example.
```
Comment (seeflood) [inline docs/en/operation/README.md:22]: It's not contain
语法不对吧

**Review by seeflood — CHANGES_REQUESTED** [review]

**Comment by seeflood** [comment]
Thanks for your contribution!
Please sign the cla first

File: docs/en/operation/README.md (hunk 3 above)
Comment (palering) [reply docs/en/operation/README.md:22]: It does not contain

File: docs/en/operation/README.md (hunk 2 above)
Comment (palering) [reply docs/en/operation/README.md:13]: 那直接点 You can start Layotto directly via executing the binary file

**Comment by palering** [comment]
> Thanks for your contribution! Please sign the cla first

In the wehsite<http://cla.sofastack.tech/>, the msg `Thank you for signing the Agreement!`  has been shown.But the status of  the mosn-bot/cla has not been updated, so what should I do now?

**Comment by seeflood** [comment]
> > Thanks for your contribution! Please sign the cla first
> 
> In the wehsitehttp://cla.sofastack.tech/, the msg `Thank you for signing the Agreement!` has been shown.But the status of the mosn-bot/cla has not been updated, so what should I do now?

Now it's updated :)

File: docs/en/operation/README.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+The advantage of smooth upgrade is that it can automatically migrate persistent connections, which can be seen in detail by clicking the above document.
+
+The options to achieve smooth upgrade are
+
+- The MOSN listens for SIGHUP, sends a SIGHUP signal to the MOSN process, and generates a new MOSN process via ForkExec.
+  
+- Restarting a new MOSN process directly. Cross-container upgrades require Operator support.
+
+- [OpenKruise hot upgrades in-place](https://mosn.io/blog/posts/mosn-sidecarset-hotupgrade/)
```
Comment (seeflood) [inline docs/en/operation/README.md:78]:
```suggestion
- [Hot upgrade using OpenKruise](https://mosn.io/blog/posts/mosn-sidecarset-hotupgrade/)
```

**Review by seeflood — CHANGES_REQUESTED** [review]

File: docs/en/operation/README.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+Layotto and MOSN are running in the same process, which can be understood as:
+
+> Layotto == MOSN + a special grpcFilter packaged together
+
+So.
+
+> replace MOSN with Layotto == replace MOSN with "MOSN + a special grpcFilter"
+
+There is no essential difference, just pay attention to the version, the versions Layotto and MOSN must correspond to each other.
```
[the remark this answers is not in the document]
Comment (palering) [reply docs/en/operation/README.md:58]: 这样如何 the versions between Layotto and MOSN must correspond to each other

File: docs/en/operation/README.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+The previously released Layotto v0.3.0 corresponds to MOSN version v0.24.1
+
+## 3. How to upgrade Layotto
+
+There are two options to upgrade.
+
+- Upgrade sidecar container using k8s native solution
+  
+- [Smooth upgrade with automatic migrate existing persistent connections](https://mosn.io/en/docs/concept/smooth-upgrade/)
```
[the remark this answers is not in the document]
Comment (palering) [reply docs/en/operation/README.md:68]: ok，这个当时看了文章中摘要的后半句

**Review by seeflood — APPROVED** [review]
lgtm

**Review by wenxuwan — APPROVED** [review]

**Pull request merged**